lib/src/lib.rs

```diff
@@ -487,42 +413,28 @@ mod test {
     }
 
     // U
-    const U: &[u8] = &hex!("fb13adeb6518cee5f88417660841142e830a81fe334380a953406a1305e8706b");
-    const ID_U: &[u8] = &hex!("a104412b");
+    const U_TV: &[u8] = &hex!("fb13adeb6518cee5f88417660841142e830a81fe334380a953406a1305e8706b");
+    const ID_U_TV: &[u8] = &hex!("a104412b");
 
     // V
-    // TODO...
+    pub const CRED_V_TV: &[u8] = &hex!("a2026b6578616d706c652e65647508a101a501020241322001215820bbc34960526ea4d32e940cad2a234148ddc21791a12afbcbac93622046dd44f02258204519e257236b2a0ce2023f0931f1f386ca7afda64fcde0108c224c51eabf6072");
 
     // W
-    const G_W: &[u8] = &hex!("ac75e9ece3e50bfc8ed60399889522405c47bf16df96660a41298cb4307f7eb6"); // TODO: update
+    pub const W_TV: &[u8] =
+        &hex!("4E5E15AB35008C15B89E91F9F329164D4AACD53D9923672CE0019F9ACD98573F");
+    const G_W: &[u8] = &hex!("FFA4F102134029B3B156890B88C9D9619501196574174DCB68A07DB0588E4D41");
     const LOC_W: &[u8] = &hex!("636F61703A2F2F656E726F6C6C6D656E742E736572766572");
 
     #[cfg(feature = "ead-zeroconf")]
     #[test]
     fn test_ead_zeroconf() {
         let state_initiator: EdhocState = Default::default();
-        let mut initiator = EdhocInitiator::new(
-            state_initiator,
-            I,
-            G_R,
-            ID_CRED_I,
-            CRED_I,
-            ID_CRED_R,
-            CRED_R,
-        );
+        let mut initiator = EdhocInitiatorState::new(state_initiator, I, CRED_I, None);
         let state_responder: EdhocState = Default::default();
-        let mut responder = EdhocResponder::new(
-            state_responder,
-            R,
-            G_I,
-            ID_CRED_I,
-            CRED_I,
-            ID_CRED_R,
-            CRED_R,
-        );
+        let mut responder = EdhocResponderState::new(state_responder, R, CRED_V_TV, Some(CRED_I));
```

Comment:
We currently instantiate the responder with `CRED_I` of the Initiator. This should ideally be `None` and the responder should fetch, for the time being through a mock up, the value of the `CRED_I` based on `ID_CRED_I`.

Reply:
Tracked in #121.